_includes/toolbar-button-ids/table-toolbar-buttons.md

```diff
@@ -17,4 +17,14 @@
 | `tablepasterowbefore`     | Pastes the row in the clipboard before the current row. {{site.requires_5_4v}} |
 | `tableprops`              | Opens the table properties dialog.                                             |
 | `tablerowprops`           | Opens the Row properties dialog.                                               |
-| `tablesplitcells`         | Splits the current merged cell.                                                |
\ No newline at end of file
+| `tablesplitcells`         | Splits the current merged cell.                                                |
+| `tableclass`              | Adds or removes pre-defined classes to the selected table. {{site.requires_5_9v}} |
+| `tablecellclass`          | Adds or removes pre-defined classes to selected cells in the table. {{site.requires_5_9v}} |
+| `tablecellvalign`         | Sets the vertical alignment of the selected cells. {{site.requires_5_9v}}       |
+| `tablecellborderwidth`    | Sets the border width of all selected cells. {{site.requires_5_9v}}             |
+| `tablecellborderstyle`    | Sets the style of border for all selected cells. {{site.requires_5_9v}}         |
+| `tablecaption`            | Toggles the caption on the selected table. {{site.requires_5_9v}}               |
+| `tablecellbackgroundcolor`| Sets the background color of the selected cells. {{site.requires_5_9v}}         |
+| `tablecellbordercolor`    | Sets the border color of the selected cells. {{site.requires_5_9v}}             |
+| `tablerowheader`          | Toggle a row between being a table header row and a table body row. {{site.requires_5_9v}}          |
```

Comment:
rephrase: remove "and" replace with "or"

Toggle a row between being a table header row or a table body row.

Reply:
Good catch, this is also an issue in the menu items as well.

Reply:
```suggestion
| `tablerowheader`          | Toggle a row between being a table header row or a table body row. {{site.requires_5_9v}}          |
```